jupyterhub/schema.yaml

```diff
@@ -1259,6 +1259,14 @@ properties:
           Configuration for a k8s ServiceAccount dedicated for use by the
           specific pod which this configuration is nested under.
         properties:
+          create:
+            type: boolean
+            description: |
+              Whether or not to create the `ServiceAccount` resource
+          name:
+            type: ["string", "null"]
+            description: |
+              Name of the `ServiceAccount` resource, if created (if `null`, a default name is used)
```

Comment:
```suggestion
              This configuration serves multiple purposes:
              - It will be the `serviceAccountName` referenced by related Pods.
              - If `create` is set, the created ServiceAccount resource will be named like this.
              - If [`rbac.enabled`](schema_rbac.enabled) is set, the associated (Cluster)RoleBindings will bind to this name.
              If not explicitly provided, a default name will be used.
```

Reply:
I've rebased this into the first commit where I changed the schema